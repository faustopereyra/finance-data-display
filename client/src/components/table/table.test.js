import { shallow } from "enzyme";
import React from "react";
import Table from "./table.component";

it("expect to render Table component", () => {
    expect(shallow(<Table />)).toMatchSnapshot();
})