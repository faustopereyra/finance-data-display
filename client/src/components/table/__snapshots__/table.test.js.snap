// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Table component 1`] = `ShallowWrapper {}`;
